CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/templates/RecordStoreQueryPlugin.java

```diff
@@ -111,13 +111,13 @@ protected RecordStore getResult() {
     protected void read(final GraphReadMethods graph, final PluginInteraction interaction, final PluginParameters parameters) throws InterruptedException {
         switch (getRecordStoreType()) {
             case GraphRecordStoreUtilities.SOURCE:
-                query = GraphRecordStoreUtilities.getSelectedVertices(graph);
+                recordStoreQuery = GraphRecordStoreUtilities.getSelectedVertices(graph);
                 break;
             case GraphRecordStoreUtilities.TRANSACTION:
-                query = GraphRecordStoreUtilities.getSelectedTransactions(graph);
+                recordStoreQuery = GraphRecordStoreUtilities.getSelectedTransactions(graph);
                 break;
             case GraphRecordStoreUtilities.ALL:
-                query = GraphRecordStoreUtilities.getAllSelected(graph);
+                recordStoreQuery = GraphRecordStoreUtilities.getAllSelected(graph);
```

Comment:
I would make this one queryRecordStore, as it is a record store representing a query, not a query of a record store.

Reply:
Still waiting on this one.